src/components/Tabs/Tabs.tsx

```diff
@@ -237,24 +197,23 @@ export const Tabs = ({
         ref={headerRef}
       >
         <ul className={styles['tabs__list']} role="tablist">
-          {getTabs().map((tab, i) => {
+          {tabs.map((tab, i) => {
             const isActive = activeIndexState === i;
             return (
               <li
                 className={clsx(
                   styles['tabs__item'],
                   isActive && styles['eds-is-active'],
                 )}
-                key={'tabs-item-' + i}
+                key={tabIds[i]}
                 role="presentation"
               >
                 <a
-                  aria-controls={tabPanelIds[i]}
```

Comment:
Note that every anchor tag now `aria-controls` the same id. The active tab panel's id.

I think it was weird how this previously pointed to ids that weren't rendered (since only one panel is actually rendered).

Reply:
In examples that I've seen of accessible tab panels, the `aria-controls` id on a tab button points to the panel that would appear if that button is pressed and the tab becomes active. https://www.w3.org/WAI/ARIA/apg/example-index/tabs/tabs-manual.html

Reply:
It is kind of weird though 🤔